app/scripts/controllers/overview.js

```diff
@@ -427,6 +426,11 @@ angular.module('openshiftConsole')
           Logger.log("routes (subscribe)", $scope.routesByService);
         }));
 
+        watches.push(DataService.watch("endpoints", context, function(endpointsData) {
```

Comment:
what advantage do we get by watching endpoints, as opposed to looking at the status of the route (which we already have) where we can know if it has been admitted by the router?